playbooks/appsemblerPlaybooks/dogwood_multiserver_deploy_cloud_sql.yml

```diff
@@ -28,7 +28,7 @@
     COMMON_ENABLE_DATADOG: False
   roles:
     - { role: swapfile, SWAPFILE_SIZE: "4GB" }
-    - mysql_init
+    - { role: mysql_init, mysql_cert_owner: "edxapp" }
```

Comment:
@mrgnr I'm a bit concern about this, because other users needs to use the cert too, like xqueue, edx notes, certificates, etc.